src/api/tests/feature_tests/features/createDeviceReferenceDataAdditionalInteractions.success.feature

```diff
@@ -0,0 +1,119 @@
+Feature: Create Device Reference Data - success scenarios
```

Comment:
Name change to 'Create Device Reference Data Additional Interactions - success scenarios'